cli/resize_a_pool.md

```diff
@@ -0,0 +1,26 @@
+# Resizing a pool, or adding/removing prefixes to the pool
+To grow or shrink a poo, you need to add prefixes to it.
+
+The type for the range to add depends on the pools' return-type: If
+the pool should return assignments, you need to add a reservation prefix
+to the pool; if the pool returns hosts, you need to add either a reservation or assignment to the pool.
```

Comment:
if it's default-type for the pool is "host" the members need to be assignments.. they cannot be reservation.